torch/onnx/_internal/exporter/_capture_strategies.py

```diff
@@ -120,9 +120,20 @@ class TorchExportStrategy(CaptureStrategy):
     def _capture(
         self, model, args, kwargs, dynamic_shapes
     ) -> torch.export.ExportedProgram:
-        return torch.export.export(
-            model, args, kwargs=kwargs, dynamic_shapes=dynamic_shapes
-        )
+        try:
+            return torch.export.export(
+                model, args, kwargs=kwargs, dynamic_shapes=dynamic_shapes
+            )
+        except torch._dynamo.exc.UserError as exc:
```

Comment:
Do we need to import `torch._dynamo.exc`?

Reply:
It looks like suggested fix only exists under UserError right now.